src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -15,21 +15,21 @@ public static partial class StreamTests
         public async static Task VerifyValueFail()
         {
             MemoryStream stream = new MemoryStream();
-            await Assert.ThrowsAsync<ArgumentNullException>(async () => await JsonSerializer.WriteAsync("", null, stream));
+            await Assert.ThrowsAsync<ArgumentNullException>(async () => await JsonSerializer.WriteAsync(stream, "", (Type)null));
```

Comment:
Fyi, with the API change, the existing test which was previously calling (object, Type, stream) API, ended up calling the (object, Stream) generic API instead since the type string was being inferred (and he last "null" parameter was set to the optional JsonSerializerOptions).

I had to change the call to specify that the "null" parameter is actually the Type parameter to correct the overload resolution.